exercises/concept/poetry-club-door-policy/.approaches/introduction.md

```diff
@@ -0,0 +1,146 @@
+# Introduction
+
+There are various ways to solve each part of Poetry Club Door Policy. A commonality between most of the parts is needing to get a character from the provided string.
+
+There are multiple ways to do this, one of which is the standard way of using `[index]` access.
+
+One other way is to use [`charAt`][mdn-char-at], which is the same as `[index]` access for most purposes.
+
+Another method is [`at`][mdn-at], which is the same as `[index]` access, except it accepts negative numbers.
+
+A negative number will count backwards from the end of the string, unlike positive numbers, which count forwards from the start.
+
+In addition, [`substring`][mdn-substring] and [`slice`][mdn-slice] can be used. These string methods are normally used to get portions of strings, rather than a single character.
```

Comment:
```suggestion
In addition, [`substring`][mdn-substring] and [`slice`][mdn-slice] can be used.
These string methods are normally used to get portions of strings, rather than a single character.
```